slave.mk

```diff
@@ -1287,7 +1287,9 @@ $(addprefix $(TARGET_PATH)/, $(SONIC_INSTALLERS)) : $(TARGET_PATH)/% : \
 	export sonic_su_dev_signing_key="$(SECURE_UPGRADE_DEV_SIGNING_KEY)"
 	export sonic_su_signing_cert="$(SECURE_UPGRADE_SIGNING_CERT)"
 	export sonic_su_mode="$(SECURE_UPGRADE_MODE)"
+ifneq ($(SECURE_UPGRADE_PROD_SIGNING_TOOL),)
```

Comment:
Since SECURE_UPGRADE_PROD_SIGNING_TOOL is a necessary when  using prod signing, I think it would be better to check SECURE_UPGRADE_MODE and keep the error (or even increase its visibility) when it equals "prod"

Reply:
https://github.com/sonic-net/sonic-buildimage/blob/9ade881f87323f4b19703a3f8b6845df4d2d57a7/rules/config#L247
How about add default value in rules/config?

Reply:
This will work